app/Http/Controllers/ControllerDash.php

```diff
@@ -399,6 +428,9 @@ public function signupForEvent(Request $request) {
                 $reg->status = 0;
                 $reg->choice_number = 1;
                 $reg->save();
+                */ // What is this for? it looks like it is for updating, but there is no such button on the frontend...
```

Comment:
I think there used to be a way to update your event registrations from the profile page. Thanks for finding this dead code (assuming @kjporter confirms), but let's remove it completely instead of commenting it out. If this is unreachable, then we shouldn't need an if statement here